src/pages/People.tsx

```diff
@@ -0,0 +1,60 @@
+import { useContext, useEffect, useState } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const People: React.FC = () => {
+  const { people, setPeopleLoader, setPeople, peopleLoader } =
+    useContext(PeopleContext);
+  const [hidePeople, setHidePeople] = useState(true);
```

Comment:
The `hidePeople` state might be redundant because its logic overlaps with `peopleLoader` and `people` states. Consider simplifying the logic by relying on `peopleLoader` and `people.length` to determine visibility.

Reply:
Thank you very much for all comments! 
Only one think about this one, actually I used this extra state to hide People before loading. Because it appears for milliseconds before loading.

Reply:
I see that it's working correctly now, although after npm start a people was shown for a milisecond before loader)